assets/js/user_activities_events/user_activities_events_table.js

```diff
@@ -91,13 +72,74 @@ export class UserActivityEventLogTable extends BaseEventLogTable {
             <UserActivityEventHistoryRow
               key={key}
               object={historicalData[key]}
+              selectModalEdit={this.selectModalEdit}
+              confirmDelete={this.confirmDelete}
             />
           ))}
         </tbody>
       </table>
     );
   }
 
+  renderEditModal() {
+    const activitiesKeys = Object.keys(this.props.userActivityTypes);
```

Comment:
Hopefully you get more inspiration around this, this was a little bit more hacky than you like